scripts/workers/code_upload_submission_worker.py

```diff
@@ -57,6 +64,43 @@ def get_volume_list():
     return volume_list
 
 
+def get_empty_volume_object(volume_name):
+    empty_dir = client.V1EmptyDirVolumeSource()
+    volume = client.V1Volume(empty_dir=empty_dir, name=volume_name)
+    return volume
+
+
+def get_config_map_volume_object(config_map_name, volume_name):
+    config_map = client.V1ConfigMapVolumeSource(name=config_map_name)
+    volume = client.V1Volume(config_map=config_map, name=volume_name)
+    return volume
+
+
+def create_config_map_object(config_map_name, file_paths):
+    # Configure ConfigMap metadata
+    metadata = client.V1ObjectMeta(
+        name=config_map_name,
+    )
+    config_data = {}
+    for file_path in file_paths:
+        file_name = os.path.basename(file_path)
+        file_content = open(file_path, "r").read()
+        config_data[file_name] = file_content
+    # Instantiate the config_map object
+    config_map = client.V1ConfigMap(
+        api_version="v1", kind="ConfigMap", data=config_data, metadata=metadata
+    )
+    return config_map
+
+
+def create_script_config_map(config_map_name):
+    submission_script_file_path = "code_upload_worker_utils/make_submission.sh"
+    script_config_map = create_config_map_object(
```

Comment:
@savish28 Please add a try except block + logging message here. Isn't this communicating with something external?

Reply:
@KhalidRmb this is creating the config map object locally and not connecting to external service at this stage, It connects with eks at a later stage where the actual job is created.

Reply:
We can take the example of V1Volume as well, all these objects Volume, ConfigMap, VolumeMounts are created as-is, although they are all in a single try-except block at the creation stage where it connects with external client.